packages/rich-text-types/src/inlines.ts

```diff
@@ -1,11 +1,9 @@
 /**
  * Map of all Contentful inline types. Inline contain inline or text nodes.
  */
-enum INLINES {
+export enum INLINES {
```

Comment:
Does this have any effects on the package exports though? Careful please not to introduce a breaking change here :sweat_smile:

Reply:
Our [public interface](https://github.com/contentful/rich-text/blob/cc26cabcc14f7f2fa2a12dbf513da195b75843a2/packages/rich-text-types/src/index.ts#L4-L5) is still the same. I guess it will only break for people importing from a sub-dir like `@contentful/rich-text-types/src/blocks` or something.